Sources/Overload/OvCore/src/OvCore/Helpers/GUIDrawer.cpp

```diff
@@ -291,6 +291,38 @@ OvUI::Widgets::Texts::Text& OvCore::Helpers::GUIDrawer::DrawSound(OvUI::Internal
 	return widget;
 }
 
+OvUI::Widgets::Texts::Text& OvCore::Helpers::GUIDrawer::DrawAsset(OvUI::Internal::WidgetContainer& p_root, const std::string& p_name, std::string& p_data, OvTools::Eventing::Event<>* p_updateNotifier)
+{
+    CreateTitle(p_root, p_name);
+
+    std::string displayedText = (p_data != "" ? p_data : std::string("Empty"));
```

Comment:
I think displayedText can be const ?

Reply:
Also performance wise, can't we just ask for p_data.empty() instead of comparing two strings ?

Reply:
Same problem all over the `GUIDrawer` (This line is actually copied from other `GUIDrawer` methods). We can open an issue later if needed.